plugins/BEdita/Core/src/Utility/Properties.php

```diff
@@ -77,13 +77,17 @@ public static function create(array $properties, array $options = []): array
 
         foreach ($properties as $p) {
             static::validate($p);
+            $options = Hash::get($p, 'options', null);
+            $options = is_array($options) ? json_encode($options) : $options;
             $property = $Properties->newEntity([
                 'name' => $p['name'],
                 'property_type_name' => $p['property'],
                 'object_type_name' => $p['object'],
                 'description' => Hash::get($p, 'description'),
                 'is_nullable' => (bool)Hash::get($p, 'is_nullable', true),
                 'read_only' => (bool)Hash::get($p, 'read_only', false),
+                'default_value' => Hash::get($p, 'default_value', null),
+                'property_options' => $options,
```

Comment:
```suggestion
            $property = $Properties->newEntity([
                'name' => $p['name'],
                'property_type_name' => $p['property'],
                'object_type_name' => $p['object'],
                'description' => Hash::get($p, 'description'),
                'is_nullable' => (bool)Hash::get($p, 'is_nullable', true),
                'read_only' => (bool)Hash::get($p, 'read_only', false),
                'default_value' => Hash::get($p, 'default_value', null),
                'property_options' => Hash::get($p, 'options', null),
```

Reply:
no need, as we removed `property_options`